src/platform.re

```diff
@@ -51,3 +51,25 @@ let version = () =>
   | Some(v) => v
   | None => raise(UnknownVersion)
   };
+
+type selectOptions('a) = {
+  ios: 'a,
+  android: 'a,
+};
+
+[@bs.deriving abstract]
+type _selectOptions('a) = {
+  [@bs.as "ios"]
```

Comment:
I think that we don't need to annotate fields if we don't want to change the fields name

Reply:
Good point. Will remove the annotations.

Reply:
@czystyl : Updated. Please re-review.